### ISSUE TITLE
Build Errors

### ISSUE THREAD

**Issue by LZL0** [JohnSully/KeyDB#35]
I hope I am not wasting your time, because I intend not to.
This is the results of the Redis tests on my Ubuntu 16.04 LTS:

```
!!! WARNING The following tests failed:

*** [err]: Command Filter REDISMODULE_CMDFILTER_NOSELF works as expected in tests/modules/commandfilter.tcl
Expected '' to be equal to '{set mykey @log}'
*** [err]: SLAVE can reload "lua" AUX RDB fields of duplicated scripts in tests/integration/replication-3.tcl
Replication not started.
```

If this is the expected result, please ignore this and close the thread.

**Comment by JohnSully** [comment]
I have not seen that failure but I will investigate.  

I'm always happy to receive issues so please don't hesitate to open them if you see something that's not working correctly.

**Comment by LZL0** [comment]
You may close this issue if you like.

**Comment by JohnSully** [comment]
I have this as part of my backlog to improve test reliability.  However it doesn't appear to be a production issue so it will be a bit behind other issues on the TODO list.

**Comment by JohnSully** [comment]
I’ve cleaned up the tests a lot in the unstable branch. If you have time to try it I would appreciate it.

**Comment by LZL0** [comment]
My apologies for the delay.

There seems to be a minor error while building.
```
networking.cpp:625:16: note: suggested alternative:
In file included from /usr/include/c++/5/random:38:0,
                 from /usr/include/c++/5/bits/stl_algo.h:66,
                 from /usr/include/c++/5/algorithm:62,
                 from server.h:55,
                 from networking.cpp:31:
/usr/include/c++/5/cmath:621:5: note:   ‘std::isinf’
     isinf(_Tp __x)
     ^
Makefile:284: recipe for target 'networking.o' failed
make[1]: *** [networking.o] Error 1
make[1]: Leaving directory '/var/keydb-unstable/KeyDB-unstable/src'
Makefile:6: recipe for target 'all' failed
make: *** [all] Error 2
```

**Comment by JohnSully** [comment]
What is your GCC/clang version?

**Comment by JohnSully** [comment]
This is specific to GCC v5.  I will support GCC versions capable of C++14 so I've fixed this with the change: 6cf85d1d01e56de0bb6e5d34590623cd8dabd32b

**Comment by JohnSully** [comment]
I've verified everything compiles and all tests pass on Ubuntu 16.04.  If you have any more issues please reopen.
